templates/channel.jade

```diff
@@ -38,7 +38,7 @@ html(lang="en")
           #main.row
             #chatwrap.col-lg-5.col-md-5
               #chatheader
-                i#userlisttoggle.glyphicon.glyphicon-chevron-up.pull-left.pointer(title="Show/Hide Userlist")
+                i#userlisttoggle.glyphicon.glyphicon-chevron-right.pull-left.pointer(title="Show/Hide Userlist")
```

Comment:
Not opposed to changing this chevron (and maybe I should actually make it switch directions with JS), but can you check how easily it could be aligned better on the other themes?  It looks fine on Modern (maybe because it's far away from the centered text) but on Light it looks strangely misaligned.